### ISSUE TITLE
error with Lint/SuppressedException in rubocop 0.84 

### ISSUE THREAD

**Issue by doudou** [rubocop-hq/rubocop#8013]
Rubocop 0.84 reported an internal error within Lint/SuppressedException on https://raw.githubusercontent.com/rock-core/tools-syskit/master/lib/syskit/models/data_service.rb

--------

## Expected behavior

`rubocop --only Lint/SuppressedException` should finish without errors

## Actual behavior

Under 0.84.0 (using Parser 2.7.1.2, rubocop-ast 0.0.3, running on ruby 2.5.1 x86_64-linux-gnu)

`rubocop --only Lint/SuppressedException -d` reports

~~~
An error occurred while Lint/SuppressedException cop was inspecting /home/tools-syskit/lib/syskit/models/data_service.rb:354:12.
bad value for range
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/cop/lint/suppressed_exception.rb:85:in `comment_between_rescue_and_end?'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/cop/lint/suppressed_exception.rb:72:in `on_resbody'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/cop/commissioner.rb:57:in `block (2 levels) in trigger_responding_cops'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/cop/commissioner.rb:136:in `with_cop_error_handling'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/cop/commissioner.rb:56:in `block in trigger_responding_cops'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/cop/commissioner.rb:55:in `each'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/cop/commissioner.rb:55:in `trigger_responding_cops'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/cop/commissioner.rb:32:in `block (2 levels) in <class:Commissioner>'
/var/lib/gems/2.5.0/gems/rubocop-ast-0.0.3/lib/rubocop/ast/traversal.rb:176:in `block in on_case'
/var/lib/gems/2.5.0/gems/rubocop-ast-0.0.3/lib/rubocop/ast/traversal.rb:175:in `each'
/var/lib/gems/2.5.0/gems/rubocop-ast-0.0.3/lib/rubocop/ast/traversal.rb:175:in `on_case'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/cop/commissioner.rb:33:in `block (2 levels) in <class:Commissioner>'
/var/lib/gems/2.5.0/gems/rubocop-ast-0.0.3/lib/rubocop/ast/traversal.rb:107:in `on_def'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/cop/commissioner.rb:33:in `block (2 levels) in <class:Commissioner>'
/var/lib/gems/2.5.0/gems/rubocop-ast-0.0.3/lib/rubocop/ast/traversal.rb:57:in `block in on_begin'
/var/lib/gems/2.5.0/gems/rubocop-ast-0.0.3/lib/rubocop/ast/traversal.rb:57:in `each'
/var/lib/gems/2.5.0/gems/rubocop-ast-0.0.3/lib/rubocop/ast/traversal.rb:57:in `on_begin'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/cop/commissioner.rb:33:in `block (2 levels) in <class:Commissioner>'
/var/lib/gems/2.5.0/gems/rubocop-ast-0.0.3/lib/rubocop/ast/traversal.rb:99:in `on_class'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/cop/commissioner.rb:33:in `block (2 levels) in <class:Commissioner>'
/var/lib/gems/2.5.0/gems/rubocop-ast-0.0.3/lib/rubocop/ast/traversal.rb:57:in `block in on_begin'
/var/lib/gems/2.5.0/gems/rubocop-ast-0.0.3/lib/rubocop/ast/traversal.rb:57:in `each'
/var/lib/gems/2.5.0/gems/rubocop-ast-0.0.3/lib/rubocop/ast/traversal.rb:57:in `on_begin'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/cop/commissioner.rb:33:in `block (2 levels) in <class:Commissioner>'
/var/lib/gems/2.5.0/gems/rubocop-ast-0.0.3/lib/rubocop/ast/traversal.rb:157:in `on_while'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/cop/commissioner.rb:33:in `block (2 levels) in <class:Commissioner>'
/var/lib/gems/2.5.0/gems/rubocop-ast-0.0.3/lib/rubocop/ast/traversal.rb:157:in `on_while'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/cop/commissioner.rb:33:in `block (2 levels) in <class:Commissioner>'
/var/lib/gems/2.5.0/gems/rubocop-ast-0.0.3/lib/rubocop/ast/traversal.rb:57:in `block in on_begin'
/var/lib/gems/2.5.0/gems/rubocop-ast-0.0.3/lib/rubocop/ast/traversal.rb:57:in `each'
/var/lib/gems/2.5.0/gems/rubocop-ast-0.0.3/lib/rubocop/ast/traversal.rb:57:in `on_begin'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/cop/commissioner.rb:33:in `block (2 levels) in <class:Commissioner>'
/var/lib/gems/2.5.0/gems/rubocop-ast-0.0.3/lib/rubocop/ast/traversal.rb:14:in `walk'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/cop/commissioner.rb:44:in `investigate'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/cop/team.rb:124:in `investigate'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/cop/team.rb:112:in `offenses'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/cop/team.rb:44:in `inspect_file'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/runner.rb:295:in `inspect_file'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/runner.rb:245:in `block in do_inspection_loop'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/runner.rb:277:in `block in iterate_until_no_changes'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/runner.rb:270:in `loop'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/runner.rb:270:in `iterate_until_no_changes'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/runner.rb:241:in `do_inspection_loop'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/runner.rb:122:in `block in file_offenses'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/runner.rb:146:in `file_offense_cache'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/runner.rb:120:in `file_offenses'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/runner.rb:111:in `process_file'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/runner.rb:90:in `block in each_inspected_file'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/runner.rb:89:in `each'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/runner.rb:89:in `reduce'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/runner.rb:89:in `each_inspected_file'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/runner.rb:78:in `inspect_files'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/runner.rb:39:in `run'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/cli/command/execute_runner.rb:21:in `execute_runner'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/cli/command/execute_runner.rb:13:in `run'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/cli/command.rb:10:in `run'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/cli/environment.rb:17:in `run'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/cli.rb:65:in `run_command'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/cli.rb:72:in `execute_runners'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/lib/rubocop/cli.rb:41:in `run'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/exe/rubocop:13:in `block in <top (required)>'
/usr/lib/ruby/2.5.0/benchmark.rb:308:in `realtime'
/var/lib/gems/2.5.0/gems/rubocop-0.84.0/exe/rubocop:12:in `<top (required)>'
/usr/local/bin/rubocop:23:in `load'
/usr/local/bin/rubocop:23:in `<main>'
~~~

## Steps to reproduce the problem

Run `rubocop --only Lint/SuppressedException` on https://raw.githubusercontent.com/rock-core/tools-syskit/master/lib/syskit/models/data_service.rb

## RuboCop version

```
$ rubocop -V
0.84.0 (using Parser 2.7.1.2, rubocop-ast 0.0.3, running on ruby 2.5.1 x86_64-linux-gnu)
```

**Comment by koic** [comment]
Dup with #8008 and #8011, and fixed by #8010. Thank you.

**Comment by doudou** [comment]
Ooops. Sorry.
